homeworks/kateryna.chepurna_niampire/homework_markup_1/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Page Title</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="style.css" />
+    <link href="//fonts.googleapis.com/css?family=Open+Sans:300,400,600,700,800" rel="stylesheet" type="text/css">
+</head>
+<body>
+<header class="heading">
+    <div class="container">
+        <div class="heading-grid">
+            <a href="#">
+                <img class="logo" src="images/logo.png" title="dreams">
+            </a>
+            <nav class="nav">
+                <div class="nav-menu">
+                    <ul>
```

Comment:
a - class must be nav-menu__list--link if you use BEM

Reply:
fixed.
You may close this pull request because, homework_markup_1  was merged.